mantidproject/mantid
# Adding plotting to Muon Analysis 2

**Description of work.**

This PR adds plotting to muon analysis 2. This is currently simple plotting of a single workspace. It also updates the multiplotting widget to work in workbench.

**Report to:** Peter

**To test:**
  * Ideally test in both workbench and mantidplot
  * Open Muon Analysis 2, when you load some data a plot window should pop up.
  * If you then do a fit on this data the fit should be added to the plot. 
  * Loading multiple runs should add all the runs to the graph and a fit on any of these graphs should then be added automatically. 
  * Check that plotting works correctly in frequency domain analysis
  * In mantidplot check the elemental analysis plotting still functions correctly.



Fixes #25738  


*This does not require release notes* because this adds functionality to a new interface which is having it's release notes done separately.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
